.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,32 @@
+name: CI
```

Let's use the same name as we have in loopback-next please, for consistency.

See https://github.com/strongloop/loopback-next/blob/8077f8e5954e9d0b638808a1efb8b703f9061b5f/.github/workflows/continous-integration.yaml.

```suggestion
name: Continous Integration
```

---

Done